README.md

```diff
@@ -1 +1,22 @@
-# hw01
\ No newline at end of file
+---
+output:
+  html_document: default
+  pdf_document: default
+---
+# hw01
+## About me
+## My name is Jared and I'm from Colorado. 
+### My Hobbies
+* I play **guitar** and love Rock and Alternative music
+* I love science fiction and fantasy stories
+  + My favorite book is *Hitchhiker's Guide to the Galaxy*
+* I am a big fan of movies and video games
+  + Some of my favorite movies are *The Matrix* and *Monty Python and the Holy Grail*
+* I act and have been doing mainly Shakespeare plays lately
+
+#### I am majoring in TAPS and Fundementals and am a rising 2nd year in *The College*
```

Comment:
If content should be a sentence or block of text, no need to put it in a header. It's probably better as just a plain block of text. I.e.

```
I am majoring in TAPS and Fundementals and am a rising 2nd year in *The College*.

I have found Markdown and using R to write to be very awkward for this, but that might just be because I don't like talking about myself. The whole Github workflow seems unneccisarily confusing but I'm sure that will become easier with time. 

Now here is a photo of my favorite childhood animal:
```

I am majoring in TAPS and Fundementals and am a rising 2nd year in *The College*.

I have found Markdown and using R to write to be very awkward for this, but that might just be because I don't like talking about myself. The whole Github workflow seems unneccisarily confusing but I'm sure that will become easier with time. 

Now here is a photo of my favorite childhood animal: